modules/local/anicluster/aniclust/tests/main.nf.test

```diff
@@ -1,23 +1,23 @@
 nextflow_process {
 
     name "Test process: ANICLUSTER_ANICLUST"
-    script "modules/local/anicluster/aniclust/main.nf"
+    script "../main.nf"
     process "ANICLUSTER_ANICLUST"
     tag "modules"
     tag "modules_local"
     tag "anicluster"
     tag "anicluster_aniclust"
 
 
-    test("Should run without failures") {
+    test("test_anicluster_aniclust") {
```

Comment:
the test process name is printed out anyway by nf-test...recommend adding the test dataset that is being tested...something like (`bacteroides_fragilis [contigs_fa_gz]`)